src/renderer/components/cluster-manager/bottom-bar.tsx

```diff
@@ -16,6 +19,13 @@ export class BottomBar extends React.Component {
           <span className="workspace-name">{currentWorkspace.name}</span>
         </div>
         <WorkspaceMenu htmlFor="current-workspace"/>
+        <div id="support" className="flex gaps align-center box right">
+          <Icon 
+            small
+            svg="support"
```

Comment:
this icon is listed on the material-design-icons website but is not present in the npm repo. There are reports of other icons missing and this repo not being maintained. Have we considered fontawesome instead?

Reply:
It seems that we should manage latest updates of material-icons-font in our repo (quick) or consider to switch to another icon-font library with active maintainers.